blocks/columns/columns.css

```diff
@@ -410,6 +410,10 @@ div.img-col > video {
   text-decoration: underline;
 }
 
+.columns.light-blue {
```

Comment:
light-background

Reply:
Sure. Actually - light can be any colour hence mentioning the colour there as we did for other variants in styles.css.